### DISCUSSION TITLE
Add days to skip spending

### DISCUSSION THREAD

**Discussion by Wjxfi** [danilkinkin/buckwheat#81] [Ideas]
I have a budget of 1,000 every day (my currency). But each week on weekends and Sundays I do not make sense to spend (I want to skip a day or set a custom value for the day). For this reason, I specify 20 000 (per month), not 30 000. But the application shows that I spend 667 per day if I specify 20,000. This is a mistake (I spend a thousand). Do something so it can be corrected. There is no point in using the app right now

------------------------------

У меня есть бюджет в 1000 каждый день (моя валюта). Но каждую неделю по субботам и воскресеньям мне нет смысла тратить (я хочу пропустить день или установить пользовательское значение для дня). По этой причине я указываю 20 000 (в месяц), а не 30 000. Но приложение показывает, что я трачу 667 в день, если я указываю 20 000. Это ошибка (я трачу тысячу). Сделайте что-нибудь, чтобы это можно было исправить. Сейчас нет смысла пользоваться приложением.

**Comment by Juliaria08** [comment]
So, basically what you want is a setting to be able to ignore certain days when calculating the daily quota.
And you also want to be able to select days for it to skip, or set specific daily quotas?

**Comment by Wjxfi** [comment]
Yes

**Comment by danilkinkin** [comment]
Hello, sorry for not answering for a long time.

It seems to me that this is an unnecessary functionality for rather rare cases.

And how would you like to count if you nevertheless made a waste on a missed day?

**Comment by Wjxfi** [comment]
without this functionality I cannot use the application. If I remove these unnecessary days, I will not be able to record anything in them, which is what I need. The answer is that I can't.

**Comment by danilkinkin** [comment]
I move this issue to discussion because think there needs some discussion before

**Comment by Wjxfi** [comment]
Where "some discussion" @danilkinkin  ?  i really need it , without it i cannot use app .
Please, Please, do it, remember it ...
